src/commands/MurderNistery/MurderNistery.js

```diff
@@ -0,0 +1,288 @@
+        this.endGame(client, channel);
+        return;
+      }
+
+      await this.lynch(client, channel);
+      if (!client.nistery.players[client.nistery.killerPos].alive || client.nistery.deadCount >= client.nistery.players.length - 2) {
+        this.endGame(client, channel);
+        return;
+      }
+    }
+  }
+
+  async nightKiller(client, user) {
+    let messageString = "Use the emojis below to choose your victim. Also, it's pretty lame but you can press the ❌ if you don't want to kill anybody...\n";
+    const reactions = [];
+
+    client.nistery.players.forEach((player) => {
+      if (player.id === user.id || !player.alive) return;
+      messageString += player.username + ": " + player.emoji + "\n";
+      reactions.push(player.emoji);
+    });
+    reactions.push('❌');
+
+    const message = await user.send(messageString);
+    reactions.forEach(react => message.react(react));
+
+    try {
+      const reaction = await message.awaitReactions((r, u) =>
+        u.id === user.id
+        && reactions.includes(r.emoji.name)
+      , {
+        max: 1,
+        time: 30000,
+        errors: ['time']
+      });
+
+      const emoji = reaction.first().emoji.name;
+      if (emoji === '❌') return -1;
+
+      for (let i = 0; i < client.nistery.players.length; ++i)
+        if (emoji === client.nistery.players[i].emoji) {
+          client.nistery.players[i].alive = false;
+          client.nistery.deadCount++;
+          return i;
+        }
+    } catch(e) {
+      user.send("You took too long to choose your victim. Now you can't kill anybody 😤");
+      return -1;
+    }
+  }
+
+  async innoNight(user) {
+    if (!user.alive) return;
+    user.send("Please type your new death message. If you don't want to change it, answer with ❌");
+
+    try {
+      let userMessage = await user.dmChannel.awaitMessages(m => m.author.id === user.id, {
+        max: 1,
+        time: 30000,
+        errors: ['time']
+      });
+
+      userMessage = userMessage.first().content;
+      if (userMessage === '❌') return;
+      user.will = userMessage;
+    } catch(e) {
+      user.send("The night has ended. Try to type faster next time 😠");
+    }
+  }
+
+  async lynch(client, channel) {
+    // we should only accept a vote if the majority voted for that player
+    await channel.send("The sun has risen 🌄 You have now 1 minute to vote and lynch someone using the emojis below. The person with the majority of votes dies ☠️");
+
+    let message = "Voting results:\n";
+    client.nistery.players.forEach((p) => {
+      if (p.alive)
+        message += `${p.emoji} ${p.username}: 0 votes\n`;
+    });
+    message += "❌ No lynch: 0 votes";
+
+    const voteMessage = await channel.send(message);
+    client.nistery.voteMessage = voteMessage;
+
+    await client.nistery.players.forEach(async (p) => {
+      if (p.alive)
+        await voteMessage.react(p.emoji);
+    });
+    await voteMessage.react('❌');
+
+    await sleep(60000);
```

Comment:
Shouldn't this sleep value be parameterized somewhere where we can easily change it? Like on the top of the file